index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <title>Grow Megha Portfolio</title>
+  </head>
+
+  <body>
+    <header class="headercontainer">
+      <div class="logo">Megha</div>
```

Comment:
- [ ] `<div class="logo">Megha</div>` this element is supposed to be the logo in this case, so the project is more semantic let's use an `<a>` stage instead of `<div>`
```
<a href="#" class="logo">Megha</a>
```
> if you notice on most websites, the logo is usually a link, so you can go back to the homepage by clicking it. That kind of UX convention makes the navigation on the site more pleasant.

Reply:
- [ ] [REQUIRED] As the previous reviewer mentioned, your logo should be clickable, so use the `<a>` tag instead of  `<div>` in order to follow the standard for the convention, semantics, and user experience(UX) reasons. Please fix it as the reviewer requested.